app/window.go

```diff
@@ -235,6 +236,13 @@ func (w *Window) SetCursorName(name pointer.CursorName) {
 	})
 }
 
+// SetColor changes the current navigation/status bar color.
+func (w *Window) SetColor(top, bottom color.RGBA) {
```

Comment:
SetColor is not specific enough, SetNavigationColor and SetStatusColor are better. I suspect you added just one method because it's tedious to add new options. See my PR comment for a suggestion.

Reply:
Exactly. `SetColor` is not good, but `SetNavigationColor` and `SetStatusColor` seems worse. 🤦‍♂️
I think the `SetOptions` makes sense.